lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -29,7 +29,7 @@ def source_dir
 
 def fixtures(category)
   begin
-    fixtures = YAML.load_file(".fixtures.yml")["fixtures"]
+    fixtures = YAML.load_file( ENV['FIXTURES_YML'] || '.fixtures.yml' )['fixtures']
```

Comment:
@DavidS: I'd also like to point out that a major overhaul of puppetlabs_spec_helper is probably going to cause issues for people who got overly creative with it and may delay the public roll out of a new version if past experience is anything to go by.